### PR TITLE
Remove maybe from RSVP response choices

### LINKED ISSUE #165: Remove Maybe option from RSVP selection
As we will be using the website going forward, let's remove the ambiguity of Maybe from RSVP options.

I'm gonna take this one. Will be a good way to dig into the codebase.

### PR DESCRIPTION
**Pull request by alysivji**
Closes #165 

I think this will do the job, but I can't seem to get the development environment setup to run tests. Would like to get eyes on the PR to make sure I'm on the right track.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,12 +11,23 @@ build:
...
+	docker-compose exec web python manage.py migrate
+
+migration:
+	docker-compose exec web python manage.py makemigrations --name $(name) $(app)
+
+test:
+	docker-compose exec web python manage.py test $(module)
 
-setup: setup_env build run_db setup_db
+setup: setup_env build run migrate
```
Comment (alysivji) [inline Makefile]: `run_db` and `setup_db` are not in the Makefile

File: chipy_org/apps/meetings/models.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
 from __future__ import unicode_literals
-from django.utils import timezone
+
+import datetime
```
Comment (alysivji) [inline chipy_org/apps/meetings/models.py]: There is a reference to `datetime.timedelta` but `datetime` was not imported

File: docker-compose.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ services:
     volumes:
       - pg_data:/var/lib/postgresql/data
     ports:
-      - "5432"
+      - "9432:5432"
```
Comment (alysivji) [inline docker-compose.yml]: keep database port the same to make containerized development easier

Comment (JoeJasinski) [reply docker-compose.yml]: Might consider leaving this to the docker_compose.override.yaml to allow people to choose their DB port without changing tracked code, in the effort to expose as few ports as possible by default.

**Comment by JoeJasinski** [comment]
I made one comment, but looks good.  Regarding the tests, I have a PR https://github.com/chicagopython/chipy.org/pull/170 open for that, in which I cherry-picked tests over from my python3/django1.11 branch.  I also setup pytest there.  Hopefully the test environment will be easier to setup after that's merged.

File: chipy_org/templates/_rsvp.html [hunk 4]
```diff
@@ -5,14 +5,14 @@
...
+      <p style="color: red">Registration for this event will close on
           {{ curr_meeting.reg_close_date|date:"l F j" }} at
           {{ curr_meeting.reg_close_date|date:"g:i a" }}
       </p>
   {% endif %}
   {% endif %}
+  <!-- TODO make RSVP choices read from choices -->
   <a class="btn btn-mini rsvp" data-response="Y">Yes</a>
-  <a class="btn btn-mini rsvp" data-response="M">Maybe</a>
   <a class="btn btn-mini rsvp" data-response="N">No</a>
```
Comment (alysivji) [inline chipy_org/templates/_rsvp.html]: I will create an issue about this so I can pair with somebody to solve it at the sprint.

Comment (alysivji) [reply chipy_org/templates/_rsvp.html]: More details: this should be reading from the CHOICES variable.

**Comment by alysivji** [comment]
My `Makefile` has some extra stuff. I can rebase once we merge in @JoeJasinski's changes.

File: Makefile (hunk 1 above)
Comment (rebeldroid12) [reply Makefile]: good! all the options!

File: chipy_org/templates/_rsvp.html [hunk 5]
```diff
@@ -5,14 +5,14 @@
...
   {% else %}
   {% if curr_meeting.reg_close_date %}
-      <p style="color: red">Registration for this event will close on 
+      <p style="color: red">Registration for this event will close on
           {{ curr_meeting.reg_close_date|date:"l F j" }} at
           {{ curr_meeting.reg_close_date|date:"g:i a" }}
       </p>
   {% endif %}
   {% endif %}
+  <!-- TODO make RSVP choices read from choices -->
```
Comment (rebeldroid12) [inline chipy_org/templates/_rsvp.html]: can this get an issue created and have it put in that TODO comment so someone could take it on?

**Comment by raymondberg** [comment]
@alysivji How are you feeling about this branch? Ready to rebase?

**Comment by alysivji** [comment]
I was gonna make this change with one of the sprint participants, but that didn't work out. Will rebase and finish this PR over the long weekend.

**Comment by raymondberg** [comment]
Just checking in on this one again in our latest sweep. What are you thinking?

**Comment by alysivji** [comment]
I would like to get this PR complete. Not sure if my development environment is still up, but I'm sure it's fine.

Aiming to get back to this over the thanksgiving break

**Comment by elmq0022** [comment]
Just checking in on this one.  Anything outstanding or do we just need to rebase and merge?   Thanks!

**Comment by alysivji** [comment]
I need to update a template as well. I"m gonna join y'all at the next project night to get this done

**Comment by elmq0022** [comment]
Sounds great.  Looking forward to meeting you.

**Comment by alysivji** [comment]
Closing this, going to open another PR to fix

Rebasing is a nightmare

**Comment by elmq0022** [comment]
@alysivji if you're dropping this branch I offer to remove the maybe response as part of the updating the user login to fill out all the info in the RSVP model.  We'll probably have conflicting commits anyway.  Let me know.

**Pull request closed without merging**